assets/css/dashboard/date-picker.scss

```diff
@@ -1,9 +1,43 @@
 @import "react-calendar/dist/Calendar.css";
 
+.date-picker {
```

Comment:
This and `.time-picker` seem to be duplicates of each other, which is quite a lot of lines to duplicate. Can we extract the common styles into a general "picker" class, maybe?